ru/mail/chrome/communicator/utilityOverlay.dtd

```diff
@@ -52,7 +52,7 @@
 <!ENTITY pac.label                                      "Онлайн (прокси: авто)">
 <!ENTITY pac.accesskey                                  "a">
 
-<!ENTITY proxy.label                                    "Конфигурация прокси">
+<!ENTITY proxy.label                                    "Конфигурация прокси…">
```

Comment:
With respect to the translations, the shorter and most obvious translation should win. We want to make it easy on the people, I'm not too hung up on what Micro soft considers "correct", so long as it is obvious.  

We don't want to use slang, of course. 

Both Outgoing Server and Send Server refer to the same thing. "Send Server" is probably easier for people.  


Generally...

If ... is NOT used, then the command will run immediately.
If ... IS used, then another window or panel will open, requiring additional steps by the end user. 

In the attached screenshot, Message Filters ... is correct, because the command doesn't immediately run, it opens a panel.

However, all of the indexing sub menus are incorrect. Those commands will run immediately, and as such, we should remove the ... 

<img width="481" alt="screen shot 2016-08-27 at 7 57 14 am" src="https://cloud.githubusercontent.com/assets/7443120/18028156/e8f65550-6c2b-11e6-8a66-7fb77a4894d5.png">